src/relay/index.ts

```diff
@@ -0,0 +1,74 @@
+import 'isomorphic-fetch';
+import * as _ from 'lodash';
+import * as BigNumber from 'bignumber.js';
+import {StandardRelayerApi, SignedOrder} from '../types';
+
+export class Relay {
+    private url: string;
+    private version: number;
+    constructor(url: string, opts?: StandardRelayerApi.RelayOpts) {
```

Comment:
All of the args in this class are part of the public interface and need assertions.